### PR TITLE
Add license key in __openerp__.py

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

**Comment by tafaRU** [comment]
@yvaucher I guess the same kind of PRs will be done for the 7.0 branches, isn'it?

**Comment by yvaucher** [comment]
@tafaRU I'm not going to do it against 7.0 branches. I assume those updates will be useful in future due to change of license of Odoo which might lead to incompatibilities with license. This will ease visually check by users and/or creation of a checker module.

Anything in 7.0 force the use of license compatible with AGPL as AGPL is the license of Odoo core 7.0

File: __unported__/l10n_it_DDT_webkit/__openerp__.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
     """,
     'author': 'Agile Business Group',
     'website': 'http://www.agilebg.com',
+    'license': 'AGPL-3',
```
Comment (yvaucher) [inline __unported__/l10n_it_DDT_webkit/__openerp__.py]: GPL-3

File: __unported__/l10n_it_DDT_webkit/__openerp__.py [hunk 2]
```diff
@@ -31,6 +31,7 @@
     """,
     'author': 'Agile Business Group',
     'website': 'http://www.agilebg.com',
+    'license': 'GPL-3 or any later version',
```
Comment (max3903) [inline __unported__/l10n_it_DDT_webkit/__openerp__.py]: Why GPL for this one ?

Comment (yvaucher) [inline __unported__/l10n_it_DDT_webkit/__openerp__.py]: I setted the license refering to what is in header

    # it under the terms of the GNU General Public License as published by
    # the Free Software Foundation, either version 3 of the License, or
    # (at your option) any later version.

**Comment by nbessi** [comment]
:+1:

File: __unported__/l10n_it_DDT_webkit/__openerp__.py (hunk 2 above)
Comment (tafaRU) [inline __unported__/l10n_it_DDT_webkit/__openerp__.py]: @yvaucher it sounds strange this module has a license different from AGPL-3. Maybe it was only a oversight, I'll wait for @eLBati opinion considering he is author.

Comment (eLBati) [inline __unported__/l10n_it_DDT_webkit/__openerp__.py]: Header says GPL due to my inattention (wrong copy and paste).
I made https://github.com/yvaucher/l10n-italy/pull/1

**Comment by yvaucher** [comment]
Thanks @eLBati I merged you changes.

**Comment by tafaRU** [comment]
:+1:

**Comment by max3903** [comment]
:+1:

**Pull request merged**